README.md

```diff
@@ -60,10 +60,12 @@ NOT REQUIRED:
     - Description for the launch
 --variable RP_LAUNCH_ATTRIBUTES:"RF tag_name:tag_value"
     - Space-separated list of tags/attributes for the launch
---variable RP_TEST_ATTRIBUTES:"Long"
+--variable RP_TEST_ATTRIBUTES:"key:value"
     - Space-separated list of tags/attributes for the tests
---variable RP_LOG_BATCH_SIZE:"10"
+--variable RP_LOG_BATCH_SIZE:"Long"
```

Comment:
Long?

Reply:
Well, since there is no number types in Python, yes, it might be big enough. Actually I took it from 'RP_TEST_ATTRIBUTES' value, thought it was mixed up.

What else it could be "Number"?

Reply:
[Number types in Python.](https://www.tutorialsteacher.com/python/python-number-type#:~:text=Python%20includes%20three%20numeric%20types,%2C%20float%2C%20and%20complex%20number.) Just leave that line without changes.

Reply:
Okay